test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt

```diff
@@ -347,7 +348,56 @@ class QueryTests {
     }
 
     @Test
-    fun composedQuery_withDescriptorAndQueryAgain() {
+    fun composedQuery_withDescriptors() {
```

Comment:
Seems to be missing quite a lot of tests here. I would expect tests for each supported type pr. aggregate. Boundary tests, what happens on overflows and so on.

Reply:
Yeah, I was still devising the exhaustive testing strategy. There should be quite a lot more tests now.